pages/matchs/index.tsx

```diff
@@ -0,0 +1,101 @@
+
+import { Button } from '@chakra-ui/react';
+import { DeleteIcon, SettingsIcon } from '@chakra-ui/icons'
+import type { NextPage } from 'next';
+import { getSession, useSession } from "next-auth/react";
+import { useRouter } from 'next/router';
+import Image from 'next/image';
+import styles from './matchs.module.css';
+
+const Menu: NextPage = () => {
+
+  const { data: session } = useSession();
+  const router = useRouter();
+
+  if(!session) {
+    <p>Loading</p>
+  }
+
+  return (
```

Comment:
The component should be clean, all the layout must be created in the components.
Check the profile page.

Reply:
Ready